docs/source/start.rst

```diff
@@ -257,20 +257,20 @@ gates from the ``pyquil.gates`` module, which allows us to add operations to our
 .. note::
 
     PyQuil also provides a handy function for you to ensure that a local qvm and quilc are currently running in
-    your environment. To make sure both are available you import `from pyquil.api import local_qvm` and then run
-    `local_qvm()`. This will start a qvm and quilc instances using subprocesses if they have not already been started.
+    your environment. To make sure both are available you execute ``from pyquil.api import local_forest_runtime`` and then use
+    :py:func:`~pyquil.api.local_forest_runtime()`. This will start qvm and quilc instances using subprocesses if they have not already been started.
```

Comment:
lets just change the language here and raise a warning (as suggested by @notmgsk and @appleby). otherwise I think this PR is nearly ready to go

Reply:
Re the warning: That would mean the processes are started, then wait for maybe two or three seconds, and then check if they exited and raise a warning if necessary? Or should that be a background task and the context manager immediately yields back to the user, while there may appear a warning later?

Reply:
@notmgsk @karalekas Let me know if you have suggestions regarding changing the langauge.

Would you like to revert `execute` back to `import` and `use` to `run`?

Reply:
Looks good to me.

Reply:
Ok cool, resolving this 👍